src/yukon/frontend/src/components/Home/PlugAndPlayTable.vue

```diff
@@ -6,17 +6,17 @@
  -->
 
 <template>
-  <div class="mt-0">
+  <div class="mt-0" style="font-family: Roboto, Monospace; display: none;">
     <div v-if="error === ''">
       <input v-model="search" ref="textSerarch" class="form-control col-sm-3" type="text"
        placeholder="Search PnP Table" aria-label="Search">
 
       <table id="plugAndPlayTable" ref="plugAndPlayTable"
-       class="table table-striped table-sm table-borderless">
+       class="table table-striped table-sm table-borderless" summary="Plug-and-Play table">
         <thead>
           <tr>
-            <th>Node Id</th>
-            <th>Unique Id</th>
+            <th scope="nodeId">Node Id</th>
+            <th scope="uniqueId">Unique Id</th>
```

Comment:
Minor nit: we generally spell ID in capitals instead of Id.

Reply:
Done.